D8CORE-3957: fix the padding right to only appear on the min of lg breakpoint

jenbreese opened the pull request:
# READY FOR REVIEW

# Summary
- Fix the event cards when it is a single item in a paragraph row. 
- This piece is to make sure the padding-right only appears on the cards when in  lg breakpoint.

# Review By (Date)
- ASAP

# Urgency
- High

# Steps to Test

1. Pull in this change.
2. And the Events one https://github.com/SU-SWS/stanford_events/pull/63 
2. Clear caches.
3. Create content. Put event cards in one row.
4. Put event cards in another row with something else. 
5. View the page on responsive.
6. Verify the cards are full width with even padding on the sides and the cards have space between.

# Affected Projects or Products
- Events
- SBT, SOE, SAA themes
- Responsive only.
- https://github.com/SU-SWS/stanford_events/pull/63

# Associated Issues and/or People
- D8CORE-3957
- @katrialesser 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

pookmish commented on src/scss/components/paragraphs/_paragraph--events.scss:
```diff
@@ -1,13 +1,15 @@
-        justify-content: left;
+@include grid-media-min('lg') {
+  .stanford-events {
+    &.cards {
+      .views-view-grid {
+        > .flex-container {
+          justify-content: left;
 
-        > div:not(:last-of-type) {
-          @include responsive-spacing('padding-right', 1);
+          > div:not(:last-of-type) {
+            @include responsive-spacing('padding-right', 1);
+          }
```
This style shouldn't live in stanford_basic. it should live in the events module

jenbreese replied:
ok. i'll move it.

jenbreese commented:
I removed the noted code to the Events module and removed the scss file. I moved the grid alignment code to the events module as well. lib/scss/components/event-card/stanford_events.event-card.scss

pookmish commented:
I'm going to roll this into the `8.x-5.x` branch since it's not necessary for a hotfix at this time.

jenbreese commented:
Thank you!
